SPEC.md

```diff
@@ -97,6 +97,12 @@ Headers that start with the `Nexus-Callback-` prefix are expected to be attached
 the handler. The callback request must strip away the `Nexus-Callback-` prefix. E.g if a Start Operation request
 includes a `Nexus-Callback-Token: some-token` header, the callback request would include a `Token: some-token` header.
 
+The `Operation-Timeout` header field can be added to inform the handler how long the caller is willing to wait for an
+operation to complete. This is distinct from the more general `Request-Timeout` header which is used to indicate the
+timeout for a single HTTP request. If the `Operation-Timeout` is shorter than the `Request-Timeout`, then
+`Request-Timeout` will be truncated to this value. Format of this header value is number + unit, where unit can be `ms`
+for milliseconds, `s` for seconds, and `m` for minutes.
```

Comment:
Not sure what you mean by "truncated", I this can be left up to the user and not put it into the framework.
In the Go SDK (the only implementation of this spec today) the client sets the `Request-Timeout` to the context's timeout, not the other way around, which I think makes sense.

```suggestion
timeout for a single HTTP request. Format of this header value is number + unit, where unit can be `ms`
for milliseconds, `s` for seconds, and `m` for minutes.
```